app/models/project.rb

```diff
@@ -374,4 +377,8 @@ def has_minimum_goal_for_video?
   def reject_project_image(attributes)
     new_record? && attributes[:original_image_url].nil?
   end
+
+  def can_has_associated_plans?
```

Comment:
Can u give another example which fits in this case, @marclerodrigues ? Sounds a bit weird for me too, but I can't think in other name.